fragments/master.sh

```diff
@@ -211,9 +211,55 @@ start_k8s(){
 
 }
 
+pull_addons_image(){
```

Comment:
Taken out. I wanted to make sure that once the the api server responds with 200 we start deploying addons asap, but I think at that point most of the layers will be available.